### PR TITLE
Landing page: Added The footer and fixed issues

### LINKED ISSUE #66: Build responsive components for WhyTechmart section and 3 easy step section of Landing Page
Title
-----

Build responsive components for WhyTechmart section and 3 easy step section of Landing Page

Summary
-------

Building the whytechmart and 3 easy step section of the landing page would allow users get insightful information about Techmart

Environment
----------
[Mobile, Tablet, and Web]

Steps to reproduce
------------------
- create a reusable component called InfoSection with the header, subheader, and children props 
- create the whyTechmart section using the InfoSection component
- create the 3 easy step section using the InfoSection component
- Add sections to the landing page

Expected result
---------------
- WhyTechmart component based on the Figma design would be designed and rendered on the landing page 
- The landing page can be accessed using the homepage route.

Current result
--------------
- The landing Page has not been created

### PR DESCRIPTION
**Pull request by Basit46**
## 🛠️ Fixes Issue

Closes #66 

## 👨‍💻 Changes proposed

- Developed footer section of landing page

- Mobile responsive

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers
I fixed the earlier merged issue I encountered 

## 📷 Screenshots

footer

### CODE REVIEW

**Comment by ingressive4good** [comment]
@Basit46 please fix the conflict on this PR

**Comment by codewithmide** [comment]
@Basit46 please don't use develop as your branch name. It is confusing, kindly delete or rename it

File: src/component/footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import "./style.css";
+import facebook from "./facebook.png";
+import twitter from "./twitter.png";
+import instagram from "./instagram.png";
+import linkedin from "./linkedin.png";
```
Comment (t-bello7) [inline src/component/footer/Footer.jsx:6]: Kindly move these images to the image folder.

**Comment by dprincecoder** [comment]
This file can’t be merged right now
Why:
You have a conflicts to resolve. 
proTip: To resolve, first if you forked the repo then sync it with i4g repo so everything is up to date 
Second: checkout to you the develop branch and do git pull. 
third: checkout to a new branch called resolve conflicts, then if you can copy the code you already did in old branch that conflicts then do else you can code afresh, when done, checkout yo this branch that has conflicts then merge this new branch called resolve conflicts you created earlier then you can now see all conflicting files to resolve it. 
To make simple in vscode open app files showing the conflicts then hovering on them will ask you to either accept incoming changes, keep both Changes or compare changes here click on accept incoming changes  after all this, you will see a button that say mark as resolve click it, do so in all conflicting files, you should be good, then commit your changes

File: src/component/footer/style.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    transform: translateX(0%);
+  }
+  .section1 {
+    position: absolute;
+    top: 122px;
+    left: 20px;
+    right: 0px;
+    bottom: 0px;
+  }
+}
```
Comment (t-bello7) [inline src/component/footer/style.css:109]: The style guideline suggests you use tailwind.css

**Review by t-bello7 — COMMENTED** [review]
You're almost there @Basit46. Kindly pull the develop branch changes from the main repository to yours and fix conflict issues. Also, the landing page components have all been added to the landing page. Your code provides a completion to it. 👍🏽

**Comment by Basit46** [comment]
@t-bello7  working on it

**Comment by dprincecoder** [comment]
@Basit46 toy have little conflicts to fix

File: src/component/footer/Footer.jsx (hunk 1 above)
Comment (Basit46) [reply src/component/footer/Footer.jsx:6]: @t-bello7 The reason I used Vanilla CSS for the remainder was that the Design had some positioning  while being responsive, so I did that to make it more understandable for me and easy to fix for others in the future.

**Comment by dprincecoder** [comment]
@Basit46 look into this

**Comment by t-bello7** [comment]
@Basit46 Are you till working on these task ?

**Comment by Basit46** [comment]
@t-bello7 My system has been giving me performance issues and I don't know when it will be stable, Kindly assign to someone else. Thanks

**Pull request closed without merging**